src/Ssh/Arguments.php

```diff
@@ -127,31 +127,34 @@ public function withMultiplexing(Host $host)
      */
     private function generateControlPath(Host $host)
     {
+        $connectionHashLength = 16; // Length of connection hash that OpenSSH appends to controlpath
+        $unixMaxPath = 104; // Theoretical max limit for path length
+        $homeDir = getenv("HOME");
```

Comment:
Please, use https://github.com/deployphp/deployer/blob/master/src/Support/Unix.php#L18

Reply:
Thanks, didn't know about that one. Will fix.